lib/fhir_client.rb

```diff
@@ -12,6 +12,7 @@
 Dir.glob(File.join(root, 'fhir_client', 'sections', '**', '*.rb')).each do |file|
   require file
 end
+require_relative 'fhir_client/ext/model' #require first so reference and bundle can inherit
```

Comment:
I don't understand.

Reply:
If we remove multiversion loading from the start, there is an issue with superclass matching for the versioned classes of bundle and reference defined in `ext`.  This fixed it, I think it's because of the versioned model definitions occurring late in `model.rb` for fhir_client, and the versioned classes not being loaded from fhir_models.

If we load multiversion from the start, this is not an issue as we load the `model.rb` files in models that contain the versioned classes.  I've removed the line with the other reverted changes.